libloki/modules/mnemonic.js

```diff
@@ -190,3 +191,10 @@ for (var i in mn_words) {
     }
   }
 }
+
+function pubkey_to_secret_words(pubKey) {
+  return mn_encode(pubKey.slice(2), 'english')
```

Comment:
Is english going to be hardcoded?

Reply:
Well, we need a consistent pubkey to secret word mapping, so the language has to be hardcoded. If you have different languages set on your devices, you still want to read the same secret words to identify your device... Does that make sense?